JS-divide/script.js

```diff
@@ -0,0 +1,7 @@
+const numberToDivide = 3;
+if (numberToDivide % 3 === 0 && numberToDivide % 5 !== 0) {
+    console.log("Divisible by 3")
+} else if (numberToDivide % 5 === 0 && numberToDivide % 3 !== 0) {
+    console.log("Divisible by 5")
+} else if (numberToDivide % 3 === 0 && numberToDivide % 5 === 0) {
+    console.log("Divisible by 3 and 5")}
```

Comment:
you can rearange the order of if statements and then only one if statement would need to use the logical and `&&` operator. Do you have an idea on how to do it?